pandas/core/generic.py

```diff
@@ -3421,17 +3420,44 @@ def align(self, other, join='outer', axis=None, level=None, copy=True,
             "compatible" value
         method : str, default None
         limit : int, default None
-        fill_axis : {0, 1}, default 0
+        fill_axis : %(axes_single_arg)s, default 0
             Filling axis, method and limit
+        broadcast_axis : %(axes_single_arg)s, default None
+            Broadcast values along this axis, if aligning two objects of
+            different dimensions
 
         Returns
         -------
-        (left, right) : (type of input, type of other)
+        (left, right) : (%(klass)s, type of other)
             Aligned objects
         """
+    )
+
+    @Appender(_shared_docs['align'] % _shared_doc_kwargs)
```

Comment:
The doc-string for ``Panel.align`` (which uses generic) seems off, can you fix up? (prob just need a definition in ``panel.py``